### PR TITLE
refactor: extract axios api & use esm in netlify functions

### PR DESCRIPTION
**Pull request by hannoeru**

### CODE REVIEW

File: instance/axios.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Axios from 'axios';
+
+const { API_KEY, SERVICE_ID } = process.env;
```
Comment (shibe97) [inline instance/axios.js]: Probably `process` is not recognized when requesting from the browser

スクリーンショット 2021-04-15 11 59 55

This error can be seen in the netlify deploy preview.
https://deploy-preview-84--blog-microcms.netlify.app/

**Review by shibe97 — CHANGES_REQUESTED** [review]
Thanks for the pull request. It’s a nice improvement.
Please check the comments.

File: functions/search.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-require('dotenv').config();
-const { API_KEY, SERVICE_ID } = process.env;
+import dotenv from 'dotenv';
+import api from '../instance/axios';
+
+dotenv.config();
 
 // eslint-disable-next-line require-await
-exports.handler = async (event) => {
+export default async (event) => {
```
Comment (shibe97) [inline functions/search.js]: Does netlify functions support esm? Search is not working properly.

スクリーンショット 2021-04-15 18 38 35

Comment (hannoeru) [reply functions/search.js]: Umm, I will check it later.

**Review by shibe97 — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**